### PR TITLE
Cleanup Index and Version

### PR DESCRIPTION
**Pull request by wilwade**
Problem
=======
Due to the updates from #96 the index and readme were out of date and we wanted to switch to a single version for the entire spec.

Solution
========
Update the readme and index and get a clear version

Change summary:
---------------
* Remove individual spec statuses
* Move archivists to draft
* Clean up the index page
* Clean up the readme
* Add version to the home page
* Update the glossary
* Remove unneeded requirement from DSNP User Ids

### CODE REVIEW

File: pages/Reference/glossary.md [hunk 1]
```diff
@@ -6,20 +6,17 @@ menu: Reference
...
-* **Hash function** - A cryptographic function whose output is effectively unique for any given input without any information from the input being accessible in the output
 * **Hash Collision** - The vanishingly unlikely possibility of a hash providing the same output for two different inputs
-* **Producer** - Someone who creates content on a social media
-* **Prosumer** - Someone who is both a producer and a consumer, see "Consumer" and "Producer"
-* **Social Identity** - A representation of a user, bot or group online, see "User," and "Bot"
-* **Social Identity Address** - The contract address of the smart contract we use to represent a social identity in our system, see "Contract address" and "Social Identity"
+* **Hash function** - A cryptographic function whose output is effectively unique for any given input without any information from the input being accessible in the output
+* **Hash** - A string of characters generated from a hash function, see "Hash function"
+* **Identity Contract Address** - The contract address of the smart contract we use to represent a user's delegations identity in our system, see "Contract address"
```
Comment (aneyzberg) [inline pages/Reference/glossary.md:19]: A little unclear to me what we are trying to say here - are we trying to say this contract represents a `user's delegated identities`?

**Review by aneyzberg — APPROVED** [review]
lgtm barring a single comment

**Review by enddynayn — APPROVED** [review]

File: pages/index.md [hunk 2]
```diff
@@ -3,44 +3,97 @@ name: Introduction
...
-# Specification status definitions
+#### Status Definitions
 
 | Name | Description |
 | --- | --- |
-| **Draft** | Specification is posted as draft PR. It's open for comment and major changes. |
+| **Draft** | Open for comment and major changes. |
 | **Proposed** | Ready for formal review. It should be done except for minor changes. |
 | **Tentative** | This is the accepted plan. The specification should not change unless there are blocking issues. |
-| **Final** | This version will not change. Changes require a new specification process and new specification version number. |
```
Comment (enddynayn) [inline pages/index.md:35]: curios why did we remove this?

**Review by shannonwells — APPROVED** [review]
💯 🚢   🚀

File: pages/index.md (hunk 2 above)
Comment (wilwade) [reply pages/index.md:35]: This is due to moving the non-final stuff into the Draft folder. Once it is out of there it is final, so no need for the status.

File: pages/Reference/glossary.md (hunk 1 above)
Comment (wilwade) [reply pages/Reference/glossary.md:19]:
```suggestion
* **Identity Contract** - The smart contract used to store a user's delegated public addresses, see "Contract address"
```

**Review by enddynayn — APPROVED** [review]

**Pull request merged**